trio/_tests/verify_types.json

```diff
@@ -112,36 +112,6 @@
       "trio.serve_listeners",
       "trio.serve_ssl_over_tcp",
       "trio.serve_tcp",
-      "trio.testing._memory_streams.MemoryReceiveStream.__init__",
-      "trio.testing._memory_streams.MemoryReceiveStream.aclose",
-      "trio.testing._memory_streams.MemoryReceiveStream.close",
-      "trio.testing._memory_streams.MemoryReceiveStream.close_hook",
-      "trio.testing._memory_streams.MemoryReceiveStream.put_data",
-      "trio.testing._memory_streams.MemoryReceiveStream.put_eof",
-      "trio.testing._memory_streams.MemoryReceiveStream.receive_some",
-      "trio.testing._memory_streams.MemoryReceiveStream.receive_some_hook",
-      "trio.testing._memory_streams.MemorySendStream.__init__",
-      "trio.testing._memory_streams.MemorySendStream.aclose",
-      "trio.testing._memory_streams.MemorySendStream.close",
-      "trio.testing._memory_streams.MemorySendStream.close_hook",
-      "trio.testing._memory_streams.MemorySendStream.get_data",
-      "trio.testing._memory_streams.MemorySendStream.get_data_nowait",
-      "trio.testing._memory_streams.MemorySendStream.send_all",
-      "trio.testing._memory_streams.MemorySendStream.send_all_hook",
-      "trio.testing._memory_streams.MemorySendStream.wait_send_all_might_not_block",
-      "trio.testing._memory_streams.MemorySendStream.wait_send_all_might_not_block_hook",
-      "trio.testing.assert_checkpoints",
-      "trio.testing.assert_no_checkpoints",
-      "trio.testing.check_half_closeable_stream",
-      "trio.testing.check_one_way_stream",
-      "trio.testing.check_two_way_stream",
-      "trio.testing.lockstep_stream_one_way_pair",
-      "trio.testing.lockstep_stream_pair",
-      "trio.testing.memory_stream_one_way_pair",
-      "trio.testing.memory_stream_pair",
-      "trio.testing.memory_stream_pump",
-      "trio.testing.open_stream_to_socket_listener",
-      "trio.testing.trio_test",
       "trio.testing.wait_all_tasks_blocked",
```

Comment:
If anybody else is confused: `wait_all_tasks_blocked` is defined in `_core/[_generated]_run.py` but exported in `testing/__init__.py` and is removed from this file in #2733